### PR TITLE
PaulSeidel_Exercises05 (1639024365/ 6260) finished

### PR DESCRIPTION
**Pull request by paulseidel08**
Task 1 finished, Task 2 not completely

Task 1 output: "the integer at sequence[653] = 1639024365 can't be build out of the 25 Integers in front of it!"

### CODE REVIEW

File: Exercises/PaulSeidel/PSeidel_exercises_05.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+set_of_bags = set()
+dict_of_bags = dict()
+
+#print(list_of_lines)
+
+def listAllBags():
+    """takes the global list of lines that describe the bags as argument and lists every bag once in a global set"""
+
+    for i in range(len(list_of_lines)):
+        element_raw = list_of_lines[i]
```
Comment (thecodingcube77) [inline Exercises/PaulSeidel/PSeidel_exercises_05.py:94]: You should do 
```py
for element_raw in list_ofLines
    # some stuff here
```
semantically the same but you don't need the index.

File: Exercises/PaulSeidel/PSeidel_exercises_05.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+        #value1_index = first_index+1                       #for the code checking every commbination of integers only once
+        found_smth = 0                                      #boolean-like varaible for checking, if a pair of values was found to build the sum
+        for value1_index in range(first_index,last_index):  #going through the list and taking the current element as the first value for the addition
+            value1 = sequence[value1_index]                 #current value to be the first of the potential combination
+            missing_rest = sequence[counter]-value1         #the rest of the subtraction is the missing part of the combination
+            #if value1_index < last_index:
+            for value2_index in range(value1_index+1,last_index+1):
+                value2 = sequence[value2_index]         #index of the 2nd value for addition
+                if value2 == missing_rest:              #checking if the value2 equals to the missing rest
+                    found_smth += 1
```
Comment (RubelOS) [inline Exercises/PaulSeidel/PSeidel_exercises_05.py:43]: You could have negate this statement and return the solution directly, when an number is **not** being able to be built by the given requirements.

Fine solution for the fist task! Seemingly correct answer! Aweseome!

**Comment by RubelOS** [comment]
Please add your result in the title of your pullrequest!

**Review by RubelOS — CHANGES_REQUESTED** [review]
Please finish task 2 (just marking it as request changes so it is clear for everyone)

**Review by RubelOS — APPROVED** [review]
Nice solution to both tasks!

File: Exercises/PaulSeidel/PSeidel_exercises_05.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+# For the actual inputs, how many bags are inside your single shiny gold bag?
+# As usual, please list the answer as part of the PR.
+
+with open(".\data\input_bags.txt", "r") as file_bags:
+    list_of_lines = [a.rstrip(".\n") for a in file_bags.readlines()]
+
+dict_of_bags = dict()
+dict_of_values = dict()
+
+def dictAllBags():
```
Comment (RubelOS) [inline Exercises/PaulSeidel/PSeidel_exercises_05.py:84]: naming convention please!

**Pull request merged**